chalice/awsclient.py

```diff
@@ -37,6 +42,28 @@ class ResourceDoesNotExistError(Exception):
     pass
 
 
+class LambdaClientError(Exception):
+    def __init__(self, original_error, context):
+        # type: (Any, LambdaErrorContext) -> None
```

Comment:
I will check but I believe it was mypy complaining on accessing attributes that are not on Exception such as context.